src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotKeys = Object.keys(robot).reverse();
+  const robotValues = Object.values(robot).reverse();
+  const invRobot = {};
+
+  for (let i = 0; i < robotKeys.length; i++) {
```

Comment:
Instead of using separate arrays for keys and values, use a 'for...in' loop to iterate through the properties of the 'robot' object.